unabridged-works-volume-1/original/05-babylon-the-great.adoc

```diff
@@ -1554,9 +1600,9 @@ Learn, therefore, O inhabitants of the earth, great and small, meekness;
 learn righteousness; learn the fear of the Lord; kiss the Son;
 harden not yourselves against him, calling him a deceiver, a blasphemer, a seducer,
 a heretic (for what ye do to the least that appear in his life, ye do to him):
-but hearken to the word of his everlasting gospel, which saith, "`FEAR GOD,
-AND GIVE GLORY TO HIM,
-FOR THE HOUR OF HIS JUDGMENT IS COME;`" and ye cannot escape his
+but hearken to the word of his everlasting gospel,
+which saith, "`Fear God, and give glory to him, for the hour of his judgment is come;`"
```

Comment:
I wasn't sure if I should leave this capitalized!!

Reply:
Yeah, i'm not 100%, but since it's kind of randomly in the middle of a chunk, and not the first word or two words of a intro-ish section, I think it was probably meant to be emphasized.  So maybe let's add italics around the quotation, like this:

```ASCIIDOC
which saith, __"`Fear God, and give glory to him, for the hour of his judgment is come;`"__

```